web/.storybook/preview-head.html

```diff
@@ -0,0 +1,5 @@
+<style>
+  body{
+    overflow-y: auto !important;
```

Comment:
What is this?

Reply:
This is for docs page, it's using GlobalStyles which contains body overflow hidden and making the page not scrollable. therefore i added this to fix it.

Reply:
Where do we want it to scroll? The storybook UI has a navbar, action section (bottom), side panel, etc. I think it is on purpose to have the body not scrollable...

Reply:
This is used for enable page scroll for docs page. I've updated the css so that it should not effect the story preview.